src/main/java/uk/gov/hmcts/ccd/domain/service/common/CaseAccessService.java

```diff
@@ -68,6 +72,13 @@ public AccessLevel getAccessLevel(ServiceAndUserDetails serviceAndUserDetails) {
         return new HashSet<>(caseUserRepository.findCaseRoles(Long.valueOf(caseId), userRepository.getUserId()));
     }
 
+    public Set<String> getAccessRoles(String caseId) {
+        if (caseId == null || caseId.trim().length() == 0) {
+            return userRepository.getUserRoles();
+        }
```

Comment:
we could simplify this by considering that audit event's for a case cannot be null. There will be always at least on event in the case history of a case, the creation event. I'd say we can assume that's the case and remove these extra checks.

Reply:
Done